generator/test_case_generator.go

```diff
@@ -123,7 +123,11 @@ var testTypeToTestConfig = map[string][]testConfig{
 		{testDir: "../../../test/acceptance"},
 		{testDir: "../../../test/feature/windows/event_logs"},
 		{
-			testDir: "../../../test/feature/windows/userdata",
+			testDir: "../../../test/feature/windows/custom_start/userdata",
+			targets: map[string]map[string]struct{}{"os": {"win-2019": {}}},
+		},
+		{
+			testDir: "../../../test/feature/windows/custom_start/ssm_start",
```

Comment:
Since this is an exception case where we only need to test starting the agent with ssm commands, I think it is fine to only run on a single os.

Reply:
This is a sanity test the agent can start and get metrics.